wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -150,8 +150,7 @@ instance Migrate (V1 Core.Coin) V0.CCoin where
 
 instance (n ~ m, n ~ 12)
     => Migrate (Mnemonic n) (V0.CBackupPhrase m) where
-    eitherMigrate =
-        Right . V0.CBackupPhrase
+    eitherMigrate = error "TODO: This file should be removed"
```

Comment:
Same as above, we're going to tackle this one a bit later. For now, we can simply have `Mnemonic` from `BIP39` as you did, but keep the `eitherMigrate` a valid implementation.